### ISSUE TITLE
Accessibility Improvements

### ISSUE THREAD

**Issue by justenanderson-commits** [justenanderson-commits/whats-cookin#71]
- [x] Add a question to the search and filter fields. "What would you like to search for?" and "How would you like to filter the recipes?"
- [x] Interpolate the recipe name into the alt text for recipe images
- [x] Update README - "We chose neutral, high-contrast colors (and placed borders around our buttons) to improve the readability for those with some level of visual impairment."
- [x] Make buttons larger on hover?
- [x] Add borders to buttons for contrast (maybe)???
- [x] Add tab index (tabability) to all of our div buttons, etc.
- [x] App must be usable for color blind users
- [x] Must score as close as possible to 100% on Lighthouse
- [x] Add notes on README for any audits our test is failing
- [x] Ensure html is semantic (and add aria tags if absolutely necessary i.e. divs working as buttons).

**Comment by thomedpete** [comment]
The buttons seem to react when they are on hover so I don't know if we need to add another transform.
I think the borders for the buttons are good as is we got 100% on lighthouse pic attached 


![Image](https://user-images.githubusercontent.com/105405396/199851343-a5d3872b-cdb7-4f64-8738-16c043eac504.png)
